### ISSUE TITLE
Not all of your orders were able to be placed!

### ISSUE THREAD

**Issue by kangoala** [particl/particl-desktop#1792]
After putting item into shopping cart (Another listing with multiple images - peace symbol), it's not possible to submit the order. Error message: "Not all of your orders were able to be placed!" is showing up, despite only one order is in the basket. Other items can still be placed. However, only this specific item with only 2 instead of 5 items to be displayed, can not be placed

**Comment by bacoinin** [comment]
That listing is mine. I received a buy order from someone from Oklahoma/Azerbaijan. @kangoala was that you ?

Could be related to the #1786 as publishing the listing also displayed an error but the listing was published.

**Comment by kangoala** [comment]
Hi @bacoinin , yes this was me. My first order seemed to disappear (at least it doesn't show up as a requested item). So I tried to order a second time, but since then there's always this error message appearing. Since it doesn't show up within the purchase tab I neither can cancel nor proceed my initial order anymore.

**Comment by bacoinin** [comment]
I see. I accepted your order and did not get any error messages but now I can see that it is not in my Sell order list. Strange. Will try to buy my own listing now and see what happens.

**Comment by kangoala** [comment]
On my side no escrow or soft lock transaction is visible. I also never got to the point to accept any escrow, since the order disappeared already at the very beginning. It never showed up on the purchases tab as "requesting".

**Comment by bacoinin** [comment]
> On my side no escrow or soft lock transaction is visible. I also never got to the point to accept any escrow, since the order disappeared already at the very beginning. It never showed up on the purchases tab as "requesting".

You are right. I just tested buying the listing and I only get the error message and no bid_request is transmitted.

**Comment by zaSmilingIdiot** [comment]
Is there any specific error message in the logs generated by the particl-market between attempting to process the item in PD and the error being displayed?

PD won't just fail to process an item: if the bid_request is not being sent then there is a reason that the market cannot process it, and the failure to process the cart item is what is causing PD to display the error. If I can get an indication of what the error is then it helps tracking down what might be causing it.
I'd speculate and say that if the original listing message wasn't sent in its entirety, then there's a possibility that that might be contributing to the problem: perhaps the message hash doesn't match the hash of the item's details that the buyer has received. But lets rather confirm whats happening first.

**Comment by bacoinin** [comment]
This is what I get when trying to buy this incomplete listing (listed on the Open Market under the title "Another listing with multiple images" and has a peace sign as the display image with only 2/5 images properly displayed):
```
2021-07-19T18:39:58.343Z - debug: [api/commands/address/AddressListCommand] type:  0=["SHIPPING_OWN"]

2021-07-19T18:39:58.343Z - debug: [api/commands/address/AddressListCommand] type: SHIPPING_OWN === SHIPPING_BID : false 
2021-07-19T18:39:58.343Z - debug: [api/commands/address/AddressListCommand] type: SHIPPING_OWN === SHIPPING_BID : false 
2021-07-19T18:39:58.343Z - debug: [api/commands/address/AddressListCommand] type: SHIPPING_OWN === SHIPPING_BID : false 
2021-07-19T18:39:58.343Z - debug: [api/commands/address/AddressListCommand] type: SHIPPING_OWN === SHIPPING_OWN : true 
2021-07-19T18:39:58.343Z - debug: [api/commands/address/AddressListCommand] type: SHIPPING_OWN === SHIPPING_BID : false 
2021-07-19T18:39:58.343Z - debug: [api/commands/address/AddressListCommand] type: SHIPPING_OWN === SHIPPING_BID : false 
2021-07-19T18:39:58.344Z - debug: [api/commands/address/AddressListCommand] type: SHIPPING_OWN === SHIPPING_BID : false 
2021-07-19T18:39:58.344Z - debug: [api/commands/address/AddressListCommand] type: SHIPPING_OWN === SHIPPING_BID : false 

2021-07-19T18:39:58.344Z - debug: [api/commands/address/AddressListCommand] type: SHIPPING_OWN === SHIPPING_BID : false 
2021-07-19T18:39:58.344Z - debug: [api/commands/address/AddressListCommand] type: SHIPPING_OWN === SHIPPING_BID : false 
2021-07-19T18:39:58.344Z - debug: [api/commands/address/AddressListCommand] type: SHIPPING_OWN === SHIPPING_BID : false 
2021-07-19T18:39:58.344Z - debug: [api/commands/address/AddressListCommand] type: SHIPPING_OWN === SHIPPING_BID : false 
2021-07-19T18:39:58.344Z - debug: [api/commands/address/AddressListCommand] type: SHIPPING_OWN === SHIPPING_BID : false
```

**Comment by zaSmilingIdiot** [comment]
Should be fixed with this change: https://github.com/xludx/particl-market/pull/29

**Comment by zaSmilingIdiot** [comment]
Closing: fixed for mainnet release of v3.0.0
